### DISCUSSION TITLE
Sling Mentioned by Dagster Labs!

### DISCUSSION THREAD

**Discussion by hogank3** [slingdata-io/sling-cli#21] [General]
Hey everyone,

I recently stumbled upon slingdata-io (Sling) after watching one of Dagster Labs' launch week videos. In it, their CTO highlighted Sling while discussing some of the pitfalls of standalone EL tools such as Stitch or Fivetran.

From my personal experience, the shout-out from Dagster Labs is well-deserved – the Sling project seems fantastic and aligns perfectly with the modern data management challenges.

If you haven't caught the mention yet, you can find it in the video linked below. It's at the end of the "Standalone tools or embedded libraries?" segment.

🎥 [Watch the video here](https://www.youtube.com/watch?v=ylC1pL25Hj4&t=15m21s)

Keep up the great work, Sling team! Thank you for your contributions to the community.

(Note: I'm an independent user and not affiliated with either Sling or Dagster. Just very grateful for awesome open-source tools that make my life easier. 🙂)

**Comment by flarco** [comment]
Thanks for sharing the video, and for the kind words!

**Reply by banditelol** [reply]
Yup, the same here!
I'm coming here and began using sling after discovering it in Dagster's launch week talk

Thanks for the products, for now it really helps when we need to quickly move data without the need to setting up CDC or any bulkier services :)
